### DISCUSSION TITLE
Animations

### DISCUSSION THREAD

**Discussion by ETCore7** [Acly/krita-ai-diffusion#575] [Show and tell]
Some animations. 

https://github.com/Acly/krita-ai-diffusion/assets/150020319/bee305b1-2654-4460-9ddd-ae685cec4514

If you want to make your own:

- go to Live mode and click the play button
- fill your canvas with strokes using texture brushes. Vary with colors and blending modes until you get a nice starting frame
- click the record button and apply more strokes. Frames will be added to the animation timeline when finished
- when ready click the record button again. The timeline will be filled. Preview te video, change the FPS if necessary
- via File - Render Animation you can make a video. Pick the MP4 format if you want to share your video on Twitter/X
- for clear text on top of your animation put it in an extra layer, but hide it while drawing. Make it visible when the timeline is filled

https://github.com/Acly/krita-ai-diffusion/assets/150020319/3209686a-a8cd-47bd-ae38-8743d97aefde

This video was made by sliding a layer with mostly horizontal strokes, starting and ending outside the viewport.

Another trick to get a looping video is to make a start frame, then duplicate the layer and set its opacity to 0%. Record strokes on the original, the scene will gradually change. Towards the end slowly bring up the opacity of the copied original, at 100% it will produce the frame you started with.

**Comment by ETCore7** [comment]
Some with a frame.

https://github.com/Acly/krita-ai-diffusion/assets/150020319/cbc27377-7586-417a-b9b6-86cee9e41cc1

https://github.com/Acly/krita-ai-diffusion/assets/150020319/8032a2ea-2837-4692-8726-15045f317dd8

**Comment by arvidtappert123** [comment]
Here is another animation workflow, the same basic principles but I´m erasing the layers to create organic animations:
https://vimeo.com/932474556?share=copy

**Reply by ETCore7** [reply]
Interesting approach, thanks.
